src/app/buy-credits/buy-credits.component.ts

```diff
@@ -388,6 +388,16 @@ export class BuyCreditsComponent implements AfterContentInit, OnInit {
   }
 
   private createAndFinaliseTipDonation() {
+    // The donation amount to Big Give is whatever the user is 'tipping' in the Buy Credits form.
+    const donationAmount = this.calculatedTipAmount();
+
+    // If user fills Buy Credits form with a £0 tip to Big Give, then do NOT attempt to create
+    // the donation, because MatchBot will respond with an error as the minimum donation amount
+    // must be £1. DON-689.
```

Comment:
I don't think it's correct that there is an enforced £1 minimum @alihejazi – there certainly isn't on the main donate page. Why not check for <= 0?